packages/iotcc.py

```diff
@@ -40,13 +40,18 @@ class PackageClass(LiotaPackage):
     This is a sample package which creates a IoTControlCenter DCC object and registers edge system on
     IoTCC over WebSocket to acquire "registered edge system", i.e. iotcc_edge_system.
     """
+    global config_path
```

Comment:
Why is global variable at multiple locations? Why is it required in the first place?